specs/tree-view/tree-view.md

```diff
@@ -0,0 +1,195 @@
+# Tree View + Tree View Item
+
+## Overview
+
+There are technically two components being presented as part of this specification - tree view, and tree view item. 
+
+As defined by the W3C:
+> A tree view widget presents a hierarchical list. Any item in the hierarchy may have child items, and items that have children may be expanded or collapsed to show or hide the children. For example, in a file system navigator that uses a tree view to display folders and files, an item representing a folder can be expanded to reveal the contents of the folder, which may be files, folders, or both.
+
+### Background
+
+We recently created a React implementation of Tree View and Tree View Item with [PR #2510](https://github.com/microsoft/fast-dna/pull/2510). 
+
+Currently, a react implementation exists which supports a `defaultExpanded` value which sets the default expanded or collapsed state. In the event that the `defaultExpanded` value is false, but there is a selected child the tree will expand automatically. [Issue #2597](https://github.com/microsoft/fast-dna/issues/2597) requests an expanded prop to fully control the expanded/collapsed state.
```

Comment:
Are these specs intended to call out the React implementation? I'm not sure how useful this is in a spec.

Reply:
I think it’s useful in the context of this part of the template which is looking for background context to specific parts of the proposal. Currently if the control is instantiated with a nested child where the Parent tree is not expanded, we will expand the entire parent structure regardless of whether that has been requested. This control is independent of a react version, but I think in this specific case the requests serve to inform whoever builds the control which is beneficial.